components/developer/gcc-12/patches/Bug106813.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/libgo/runtime/go-signal.c b/libgo/runtime/go-signal.c
```

Comment:
Please add url here pointing to the bug report (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=106813).